src/platform/plugins/shared/saved_objects_tagging_oss/server/plugin.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { CoreSetup, CoreStart, Logger, Plugin, PluginInitializerContext } from '@kbn/core/server';
+import {
+  SavedObjectTaggingOssPluginSetup,
+  SavedObjectTaggingOssPluginStart,
+  SavedObjectsTaggingApiServer,
+} from './types';
+
+export class SavedObjectTaggingOssPlugin
```

Comment:
I am not sure that we need this glue code nowadays. If I am not mistaken, this pattern was needed when we couldn't depend on x-pack plugins. Now this isn't the case anymore since we can have an optional dependency. 

If this was the only reason for this plugin, then we should double-check with someone (maybe from core team) and potentially simplify. Wdyt?